src/box/lua/config/instance_config.lua

```diff
@@ -741,6 +741,11 @@ return schema.new('instance_config', schema.record({
             box_cfg = 'worker_pool_threads',
             default = 4,
         }),
+        tx_user_pool_size = schema.scalar({
+            type = 'number',
```

Comment:
`'integer'` fits better here -- at least a better diagnostic is reported if a non-integral number is passed.

I see, there are options, where `'number'` is used in the similar situation, but these are just mistakes.

Feel free to ignore, this is a negligible point.

Reply:
Fixed, thanks for noticing!